src/V3Stats.cpp

```diff
@@ -272,6 +351,15 @@ class StatsVisitor final : public VNVisitor {
                     m_stage, (string("Branch prediction, ") + VBranchPred(type).ascii()), count);
             }
         }
+        // Storage elements
+        if (m_registerCount != 0.0) {
+            V3Stats::addStat("Registers, count", m_registerCount);
+            V3Stats::addStat("Registers, total bits", m_registerBitCount);
+        }
+        if (m_memoryCount != 0.0) {
```

Comment:
```suggestion
        if (m_registerCount) {
            V3Stats::addStat("Registers, count", m_registerCount);
            V3Stats::addStat("Registers, total bits", m_registerBitCount);
        }
        if (m_memoryCount) {
```

But.... generally stats even print when zero so perhaps instead do it always?

Reply:
That's what I started with but the clang build failed with a boolean test on a floating point value. I borrowed this from the existing code immediately above it, but you're right, I think it's best to always report these.